recipes/pyqmix/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove all comments

Reply:
This has been resolved in the latest commit.